pkg/graph/resolvers/system_intake_actions_test.go

```diff
@@ -0,0 +1,253 @@
+package resolvers
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/guregu/null"
+
+	"github.com/cmsgov/easi-app/pkg/graph/model"
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+func (s *ResolverSuite) TestSystemIntakeRequestEditsAction() {
+	ctx := context.Background()
+	initialSteps := []models.SystemIntakeStep{
+		models.SystemIntakeStepINITIALFORM,
+		models.SystemIntakeStepDRAFTBIZCASE,
+		models.SystemIntakeStepFINALBIZCASE,
+		models.SystemIntakeStepGRBMEETING,
+		models.SystemIntakeStepGRTMEETING,
+		models.SystemIntakeStepDECISION,
+	}
+	formStepMap := map[model.SystemIntakeFormStep]models.SystemIntakeStep{
+		model.SystemIntakeFormStepInitialRequestForm: models.SystemIntakeStepINITIALFORM,
+		model.SystemIntakeFormStepDraftBusinessCase:  models.SystemIntakeStepDRAFTBIZCASE,
+		model.SystemIntakeFormStepFinalBusinessCase:  models.SystemIntakeStepFINALBIZCASE,
+	}
+	formSteps := []model.SystemIntakeFormStep{
+		model.SystemIntakeFormStepInitialRequestForm,
+		model.SystemIntakeFormStepDraftBusinessCase,
+		model.SystemIntakeFormStepFinalBusinessCase,
+	}
```

Comment:
If you want to ensure that you don't forget to add to this slice, you can directly use `model.AllSystemIntakeFormStep` - GQLGen handily creates a slice of all auto-generated enums for cases like this. That way if we add an enum from this list, our tests will properly start to fail since we won't handle it in `formStepMap`!

https://github.com/CMSgov/easi-app/blob/dea7417b4a3b64711f44b4bb7b66a37bcb214460/pkg/graph/model/models_gen.go#L1041-L1045

Reply:
TIL

Reply:
resolved in [c38f771](https://github.com/CMSgov/easi-app/pull/2129/commits/c38f7719b2cd5c17d9a5d3b6c1b2479fef146da5)